QEfficient/src/_transformers/auto.py

```diff
@@ -70,11 +70,7 @@ def tokenizer(self) -> Union[PreTrainedTokenizer, PreTrainedTokenizerFast]:
     def get_model_card_name(self) -> str:
         # FIXME: use getter
         if self.model_card_name is None:
-            # Handle when pretrained_model_name_or_path is a path and we don't know the model_card_name
-            assert not os.path.isdir(
-                self.pretrained_model_name_or_path
-            ), f"Please provide `model_card_name` argument as valid string, got {self.model_card_name}"
-            self.model_card_name = self.pretrained_model_name_or_path
+            self.model_card_name = generate_sha256_hash(self.pretrained_model_name_or_path)
```

Comment:
This should only be done when the provided `pretrained_model_name_or_path` is a local directory.

Reply:
yes. Updated it in the latest patch.

Reply:
The point was to force user to pass `model_card_name` at export stage. So that she/he can figure out where the exported/compiled `onnx/qpc` is saved under `QEFF_HOME`. 
In PR 85, we are moving towards hash always for PEFT models. Once this feature is added for `QEFFAutoModelForCausalLM` we don't need to do this.